src/components/Chat/Chat.test.js

```diff
@@ -63,20 +64,22 @@ describe('<Chat />', () => {
 
         test("should reset the header height", () => {
             const mockResetHeightMethod = jest.spyOn(Chat.prototype, "resetChatHeight");
-            render(<ThemeProvider>
-                      <Chat {...mockProps} />
-                    </ThemeProvider>);
+            render(<IntlProvider onError={jest.fn} locale="en">
```

Comment:
qq: does `IntlProvider onError={jest.fn} locale="en"` forces the language to be English or does it translate from whatever language back to English?

Reply:
In unit test it forces the language to be English.